<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CUartConsole</class>
 <widget class="QDialog" name="CUartConsole">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Console</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="ConnectLbl">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabWidgetPage2">
        <attribute name="title">
         <string>Input</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="leftMargin">
            <number>1</number>
           </property>
           <property name="topMargin">
            <number>1</number>
           </property>
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="_4">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QTextEdit" name="textEdit_in">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>60</height>
                </size>
               </property>
               <property name="accessibleName">
                <string/>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="documentTitle">
                <string/>
               </property>
               <property name="acceptRichText">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="_5">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="margin">
                <number>5</number>
               </property>
               <item>
                <widget class="QLabel" name="label">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Wait (ms) on LF</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lineEdit">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>500</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbOpen">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Open File ...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbSend">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Send Data</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pbStop">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>21</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>STOP Input stream</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QProgressBar" name="inputProgressBar">
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage1">
        <attribute name="title">
         <string>Output</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="_2">
           <property name="spacing">
            <number>6</number>
           </property>
           <property name="margin">
            <number>4</number>
           </property>
           <item>
            <widget class="QTextEdit" name="textEdit_out">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="acceptRichText">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="_3">
             <property name="spacing">
              <number>6</number>
             </property>
             <property name="margin">
              <number>4</number>
             </property>
             <item>
              <widget class="QPushButton" name="pbClear">
               <property name="minimumSize">
                <size>
                 <width>108</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>Clear Console</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pbSaveTo">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>Save to File ...</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Expanding</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>10</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QCheckBox" name="cbCRLF">
               <property name="text">
                <string>Interpret CR/LF</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabWidgetPage3">
        <attribute name="title">
         <string>Config</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>2</number>
           </property>
           <property name="margin">
            <number>2</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Baudrate :</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="baudCombo">
               <property name="currentIndex">
                <number>1</number>
               </property>
               <item>
                <property name="text">
                 <string>300</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>1200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>2400</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>4800</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>9600</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>19200</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>38400</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Start bit : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_2">
               <property name="text">
                <string>1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="margin">
              <number>2</number>
             </property>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Stop bit : </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_5">
               <property name="text">
                <string>1</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_6">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>2</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="_6">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbRelaySerialPort">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Relay to Serial Port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbRelayNetwork">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Relay to Network </string>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbClose">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pbClear</sender>
   <signal>clicked()</signal>
   <receiver>textEdit_out</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>411</x>
     <y>73</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
